server/controllers/monitoringConditions/attendanceMonitoringController.test.ts

```diff
@@ -0,0 +1,301 @@
+import { v4 as uuidv4 } from 'uuid'
+import { getMockOrder } from '../../../test/mocks/mockOrder'
+import AttendanceMonitoringController from './attendanceMonitoringController'
+import { createMockRequest, createMockResponse } from '../../../test/mocks/mockExpress'
+import HmppsAuditClient from '../../data/hmppsAuditClient'
+import RestClient from '../../data/restClient'
+import AuditService from '../../services/auditService'
+import AttendanceMonitoringService from '../../services/attendanceMonitoringService'
+import TaskListService from '../../services/taskListService'
+
+const mockConditionId = uuidv4()
+
+jest.mock('../../services/auditService')
+jest.mock('../../services/orderService')
+jest.mock('../../services/attendanceMonitoringService')
+jest.mock('../../data/hmppsAuditClient')
+jest.mock('../../data/restClient')
+
+describe('attendanceMonitoringController', () => {
+  let mockRestClient: jest.Mocked<RestClient>
+  let mockAuditClient: jest.Mocked<HmppsAuditClient>
+  let mockAuditService: jest.Mocked<AuditService>
+  let mockAttendanceMonitoringService: jest.Mocked<AttendanceMonitoringService>
+  let attendanceMonitoringController: AttendanceMonitoringController
+  const taskListService = new TaskListService()
+
+  beforeEach(() => {
+    mockAuditClient = new HmppsAuditClient({
+      queueUrl: '',
+      enabled: true,
+      region: '',
+      serviceName: '',
+    }) as jest.Mocked<HmppsAuditClient>
+    mockRestClient = new RestClient('cemoApi', {
+      url: '',
+      timeout: { response: 0, deadline: 0 },
+      agent: { timeout: 0 },
+    }) as jest.Mocked<RestClient>
+    mockAuditService = new AuditService(mockAuditClient) as jest.Mocked<AuditService>
+
+    mockAttendanceMonitoringService = new AttendanceMonitoringService(
+      mockRestClient,
+    ) as jest.Mocked<AttendanceMonitoringService>
+
+    attendanceMonitoringController = new AttendanceMonitoringController(
+      mockAuditService,
```

Comment:
bit pointless to mock AuditService but never actually verify anything with it. I see it is needed to construct the  AttendanceMonitoringController but then the controller also doesn't log anything via AuditService neither, so either log an audit or remove the dependency?